doc/source/text.rst

```diff
@@ -429,6 +429,27 @@ String ``Index`` also supports ``get_dummies`` which returns a ``MultiIndex``.
 
 See also :func:`~pandas.get_dummies`.
 
+Concatenating Series
+--------------------
+
+The method :meth:`Series.str.cat` can be used to concatenate the records of two :class:`Series`. Depending on the value given to the ``align`` keyword,
+their indexes will be aligned before concatenation (if ``align=True``) or not (if ``align=False``). As usual, alignment will expand to the union of both
+indexes, while introducing ``NaN`` for missing values in the respective other series (which can be easily handled with the ``na_rep``-keyword).
+
+If the ``align`` keyword is not passed, the method will currently fall back to the previous behavior (i.e. ``align=False``),
```

Comment:
Put this in a `.. warning:: ` directive.

Reply:
Which part? I added one starting at "If the ``align`` keyword is not passed"